Fails to compile on Windows with LLVM 21

```
F:/ik_llama.cpp/ggml/src/ggml-backend.cpp:2275:42: error: use of undeclared identifier 'first_reduce'
 2275 |         auto compute = [sched, &barrier, first_reduce] (int ith) {
      |                                          ^~~~~~~~~~~~
F:/ik_llama.cpp/ggml/src/ggml-backend.cpp:2283:31: error: use of undeclared identifier 'first_reduce'; did you mean 'last_reduce'?
 2283 |             int last_reduce = first_reduce;
      |                               ^~~~~~~~~~~~
      |                               last_reduce
F:/ik_llama.cpp/ggml/src/ggml-backend.cpp:2283:17: note: 'last_reduce' declared here
 2283 |             int last_reduce = first_reduce;
      |                 ^
2 errors generated.
[34/191] Building CXX object ggml/src/CMakeFiles/ggml.dir/iqk/fa/iqk_fa_96_96.cpp.obj
FAILED: [code=1] ggml/src/CMakeFiles/ggml.dir/iqk/fa/iqk_fa_96_96.cpp.obj
C:\PROGRA~1\LLVM\bin\CLANG_~1.EXE -DGGML_BUILD -DGGML_IQK_FLASH_ATTENTION -DGGML_SCHED_MAX_COPIES=1 -DGGML_SHARED -DGGML_USE_IQK_MULMAT -DGGML_USE_LLAMAFILE -DGGML_USE_VULKAN -DGGML_VULKAN_BFLOAT16_GLSLC_SUPPORT -DGGML_VULKAN_COOPMAT2_GLSLC_SUPPORT -DGGML_VULKAN_COOPMAT_GLSLC_SUPPORT -DGGML_VULKAN_INTEGER_DOT_GLSLC_SUPPORT -DNDEBUG -D_CRT_SECURE_NO_WARNINGS -D_XOPEN_SOURCE=600 -Dggml_EXPORTS -IF:/ik_llama.cpp/ggml/src/../include -IF:/ik_llama.cpp/ggml/src/. -IF:/ik_llama.cpp/build/ggml/src -isystem C:/VulkanSDK/1.4.335.0/include -O3 -DNDEBUG -D_DLL -D_MT -Xclang --dependent-lib=msvcrt -std=gnu++20 -Wmissing-declarations -Wmissing-noreturn -Wall -Wextra -Wpedantic -Wcast-qual -Wno-unused-function -Wunreachable-code-break -Wunreachable-code-return -Wmissing-prototypes -Wextra-semi -march=native -MD -MT ggml/src/CMakeFiles/ggml.dir/iqk/fa/iqk_fa_96_96.cpp.obj -MF ggml\src\CMakeFiles\ggml.dir\iqk\fa\iqk_fa_96_96.cpp.obj.d -o ggml/src/CMakeFiles/ggml.dir/iqk/fa/iqk_fa_96_96.cpp.obj -c F:/ik_llama.cpp/ggml/src/iqk/fa/iqk_fa_96_96.cpp
In file included from F:/ik_llama.cpp/ggml/src/iqk/fa/iqk_fa_96_96.cpp:5:
In file included from F:/ik_llama.cpp/ggml/src/.\iqk/fa/iqk_fa_templates.h:26:
F:/ik_llama.cpp/ggml/src/.\iqk/iqk_utils.h:86:23: error: overloaded 'operator|' must have at least one parameter of class or enumeration type
   86 | static inline __m512i operator|(__m512i a, __m512i b) { return _mm512_or_si512(a, b); }
      |                       ^
F:/ik_llama.cpp/ggml/src/.\iqk/iqk_utils.h:87:23: error: overloaded 'operator&' must have at least one parameter of class or enumeration type
   87 | static inline __m512i operator&(__m512i a, __m512i b) { return _mm512_and_si512(a, b); }
      |                       ^
F:/ik_llama.cpp/ggml/src/.\iqk/iqk_utils.h:88:23: error: overloaded 'operator^' must have at least one parameter of class or enumeration type
   88 | static inline __m512i operator^(__m512i a, __m512i b) { return _mm512_xor_si512(a, b); }
      |                       ^
```

```
cmake -B build -G Ninja -DGGML_VULKAN=ON -DCMAKE_PREFIX_PATH=C:\VulkanSDK\1.4.335.0 -DCMAKE_INSTALL_PREFIX=C:/ -DCMAKE_BUILD_TYPE=Release -DGGML_NATIVE=O
```

--- a/ggml/src/ggml-backend.cpp
+++ b/ggml/src/ggml-backend.cpp
@@ -2165,6 +2165,7 @@ static enum ggml_status ggml_backend_sched_compute_splits(ggml_backend_sched_t s
 
         for (auto & s : sched->statuses) s = GGML_STATUS_SUCCESS;
 
+        int first_reduce = -1;
         bool work_done = false;
 #ifdef GGML_USE_OPENMP
         //This may not be available in old OpenMP versions
@@ -2185,7 +2186,6 @@ static enum ggml_status ggml_backend_sched_compute_splits(ggml_backend_sched_t s
                 }
             }
         }
-        int first_reduce = -1;
         for (int i = 0; i < sched->n_splits; i++) {
             auto split = &sched->splits[i];
             if (split->graph.n_nodes == 1 && split->graph.nodes[0]->op == GGML_OP_REDUCE) {

